docs/src/main/paradox/keyspaces.md

```diff
@@ -0,0 +1,47 @@
+# Amazon Keyspaces
+
+@@@ warning
+
+**This project is not continuously tested against CosmosDB, and CosmosDB is not supported as part of the [Lightbend Subscription](https://www.lightbend.com/lightbend-subscription).**
+
+@@@
+
+[Amazon Keyspaces](https://docs.aws.amazon.com/keyspaces) is an Amazon service
+that provides a Cassandra-like API, but it's not real Cassandra and has
+different behaviour for some things (for example atomic persist of several
+events is not possible).
+
+Keyspaces is known not to work on akka-persistence-cassandra version 1.0.3 and
```

Comment:
there is a config workaround, but this is fine

Reply:
Not for pre 1.0 as it uses a default batch which is LOGGED but then real C* treats it as an UNLOGGED batch as all the writes are for the same partition